### PR TITLE
ci: Adjust actions and refactor mirror workflow

### PR DESCRIPTION
**Pull request by Techassi**
Fixes https://github.com/stackabletech/docker-images/issues/785, supersedes changes on https://github.com/stackabletech/docker-images/tree/feature/reusable-mirror-yaml

This PR renames and refactors a bunch of local actions. Additionally it adjusts the mirror workflow to use the adjusted actions.

### CODE REVIEW

**Review by NickLarsenNZ — APPROVED** [review]
Before merging, we need to test via `workflow_dispatch`:

- [ ] one of the existing workflows
- [ ] the mirror worflow

**Comment by NickLarsenNZ** [comment]
Note: The mirror workflow fails if the image already exists. We _could_ add a job that checks for existence. Then the other jobs skip if it exists.

**Comment by Techassi** [comment]
> Note: The mirror workflow fails if the image already exists. We _could_ add a job that checks for existence. Then the other jobs skip if it exists.

Yes, let's add such a check in a follow-up PR.

**Review by NickLarsenNZ — DISMISSED** [review]
LGTM

**Review by Maleware — DISMISSED** [review]
Just reviewed the Mirror.yaml and looks great. Thanks!

LGTM!

**Review by NickLarsenNZ — DISMISSED** [review]
LGTM

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

File: .github/actions/README.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+index tag and image manifest tag. All subsequent tasks must use these values to
+ensure consistency.
+
+Currently, bake provides the following ouput in the `bake-target-tags` file:
+
+```plain
+docker.stackable.tech/stackable/kafka:3.4.1-stackable0.0.0-dev-amd64
+```
+
+Until bake supports the ability to specify the registry, this action will retag
```
Comment (razvan) [inline .github/actions/README.md:52]:
```
➜  ~ bake --help
...
options:
  -r REGISTRY, --registry REGISTRY
                        Image registry to publish to. Default: docker.stackable.tech.
...
```

**Pull request merged**